piker/pp.py

```diff
@@ -680,10 +693,10 @@ def write_config(self) -> None:
         '''
         # TODO: show diff output?
         # https://stackoverflow.com/questions/12956957/print-diff-of-python-dictionaries
-        print(f'Updating ``pps.toml`` for {path}:\n')
-
+        log.info(f'Updating ``pps.toml`` for {path}:\n')
         # active, closed_pp_objs = table.dump_active()
         pp_entries = self.to_toml()
+        log.info(pp_entries)
```

Comment:
maybe add a header to this msg?

`log.info(f'Current positions:\n{pp_entries}')`